docs/Makefile

```diff
@@ -2,7 +2,7 @@
 #
 
 # You can set these variables from the command line.
-SPHINXOPTS           = -W -n
+SPHINXOPTS           = -W
```

Comment:
After working a little more on my napoleon/django extension, I was able to re-enable strict docs building. We do need to disable nit-picky mode, though, which is explained in the commit description.